tccli/configure.py

```diff
@@ -333,7 +337,20 @@ def _run_main(self, parsed_args, parsed_globals):
         for index, prompt_text in self.VALUES_TO_PROMPT:
             if index in config:
                 response = self._compat_input("%s[%s]: " % (prompt_text, config[index]))
-                conf_data[index] = response if response else config[index]
+                if index == OptionsDefine.Output:
+                    if not response in ['json', 'text', 'table']:
+                        print('Output format must be json, text or table. Set as default: %s\n' % config[index])
+                        conf_data[index] = config[index]
+                    else:
+                        conf_data[index] = response
+                elif index ==OptionsDefine.Region:
+                    if response:
+                        conf_data[index] = response
```

Comment:
maybe we should check the region value as well, for the best user experience.
but it can be done later